frontend/package.json

```diff
@@ -14,6 +14,7 @@
     "@shoutem/ui": "^0.23.12",
     "expo": "^32.0.0",
     "jest": "^24.1.0",
+    "lodash": "^4.17.11",
     "node-fetch": "^2.3.0",
```

Comment:
I'd recommend adding as a dev dependency [`eslint`](https://eslint.org/docs/user-guide/getting-started)

You can look at our code for how we implement eslint if you want an example. It helps automate both the detection, and also fixing, styling errors.

Reply:
We have made a team decision not to integrate this library at this point.